marklogic-data-hub-central/ui/e2e/cypress/support/pages/monitor.tsx

```diff
@@ -46,7 +46,8 @@ class MonitorPage {
     // BUG: The page it's re-rendering twice. There's no request to intercept.
     cy.wait(1000);
     cy.get(`[data-testid=${testId}-facet] input`).eq(index).should("be.visible").check();
-    cy.findByTestId("facet-apply-button").click({force: true});
+    cy.wait(1000);
```

Comment:
Why are we waiting 1 whole second after clicking a facet? Wouldn't it be better and faster to check/wait for the button to be enabled?

Reply:
Same reason as the above.